sonic_y_cable/y_cable_base.py

```diff
@@ -97,10 +98,11 @@ def check_read_side(self, port):
                 an Integer, the actual physical port connected to Y end of a Y cable which can which side reading the MUX from
 
         Returns:
-            an Integer, 1 if reading the Y cable from TOR A side(TOR 1).
-                      , 2 if reading the Y cable from TOR B side(TOR 2).
-                      , 0 if reading the Y cable from NIC side.
-                      , -1 if reading the Y cable API fails.
+            One of the following predefined constants:
+                TARGET_TORA, if reading the Y cable from TOR A side(TOR A).
+                TARGET_TORB, if reading the Y cable from TOR B side(TOR B).
```

Comment:
Seems like the parenthesized text is redundant now. Suggest removing.

Reply:
fixed